src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.WaitForChanged.cs

```diff
@@ -126,6 +126,7 @@ public void NonZeroTimeout_NoActivity_TimesOut(WatcherChangeTypes changeType, bo
         }
 
         [Theory]
+        [OuterLoop]
```

Comment:
Can you add a comment next to all of the Outerloop attributes with a short message indicating why it's being made Outerloop?  That way when someone comes along and thinks to themselves "this test is really quick, I'll just make it inner loop" they know why not to do that.

(We should probably actually modify the Outerloop attribute to have a message ctor parameter for that to be specified.)